charts/rosa-capi/README.md

```diff
@@ -0,0 +1,7 @@
+# rosa-capi 
+
+## Quick Start
+
+```
+helm template --name-template=rosa-hcp --set ocmToken=jSDFsdfsdfSDF . | oc apply -f -
```

Comment:
Is the use of `--name-template` needed?

Reply:
Not needed, but it yields better/nicer names of the resources